CHANGES.md

```diff
@@ -48,6 +48,12 @@
 
 - Change the formatting-on-save error notification to a warning notification (#472)
 
+- Bug fix: do not show "random" documentation on hover
```

Comment:
Isn't this already in the merlin change list? We usually do not replicate merlin's change list here.

Reply:
It is already in the merlin changes list, but since this bug has been reported several times in both ocaml-lsp & vscode-ocaml-platform, I thought it is useful for users to see it fixed. 

I think that merlin is ocaml-lsp's backend, but the end user shouldn't be lurking in merlin Changes to see what's new in ocaml-lsp -- maybe we could at least add a link to their Changes inside the ocaml-lsp-vendored merlin source.

Reply:
The change entry in merlin is "locate: reset global state from all entry points (#1364)", which is cryptic, isn't it?

Reply:
It is cryptic. But that can be mended on merlin's side. I'd prefer to see the links to the corresponding change list entries where the fix was done.

> I think that merlin is ocaml-lsp's backend, but the end user shouldn't be lurking in merlin Changes to see what's new in ocaml-lsp -- maybe we could at least add a link to their Changes inside the ocaml-lsp-vendored merlin source.

We could have a policy of replicating only the relevant merlin changes in our repo. The problem is that I often lack context as to what is relevant to lsp, and it is extra work. We can bring it up with the rest of the team to see what they think.